install/main.go

```diff
@@ -0,0 +1,45 @@
+package install
+
+var managers = make(map[string]Manager)
+
+func initManagers() {
+	managers["platform"] = PlatformManager{}
+	managers["npm"] = NpmManager{}
+}
+
+// Run function installs engines witg given Context
+func Run(c Context) []Result {
```

Comment:
I suggest `context` instead of `c`, e.g. use [Meaningful names and functions](https://medium.com/coding-skills/clean-code-101-meaningful-names-and-functions-bf450456d90c) in all projects.